.test-infra/jenkins/PythonTestProperties.groovy

```diff
@@ -17,5 +17,8 @@
  */
 
 class PythonTestProperties {
+  final static List<String> CROSS_LANGUAGE_VALIDATES_RUNNER_PYTHON_VERSIONS = ['3.6', '3.8']
```

Comment:
Thanks a lot!
I think we should make some constants like

```
LOWEST_SUPPORTED = 3.6
HIGHEST_SUPPORTED = 3.8
ESSENTIAL_VERSIONS = [LOWEST_SUPPORTED, HIGHEST_SUPPORTED]
```
We can also do it in another PR.

Reply:
Sure, I created PR about this; https://github.com/apache/beam/pull/13349